scripts/classes/app-binary-configurator.js

```diff
@@ -96,12 +95,14 @@ class AppBinaryConfigurator {
   }
 
   configureLaunchScreen() {
-    const launchScreen = this.getLaunchScreenUrl();
-    const resizeConfig = this.binarySettings.launchScreen;
-    const production = this.config.production;
-    const launchScreenPath = './assets/launchScreen.png';
-    console.log('Configuring launch screen...');
-    return downloadAndResizeImage(launchScreen, launchScreenPath, resizeConfig, production);
+    return Promise.all(_.map(binarySettings, (settings, platform) => {
+      const launchScreen = this.getLaunchScreenUrl(platform);
+      const resizeConfig = settings.launchScreen;
+      const production = this.config.production;
+      const launchScreenPath = './assets/launchScreen.png';
```

Comment:
I'd extract all this `asset` strings to one place. At first, just at the top of this file, then finally in some `assetsConfig` file.